task2/webpack.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = process.env.NODE_ENV === 'production'
```

Comment:
take a look at 'webpack-merge' npm module

good practice to have 3 files for webpack:

prod, dev, common 

webpack.common.js will contain common configs for both prod and dev - e.g. loaders